.github/workflows/test.yml

```diff
@@ -0,0 +1,50 @@
+name: ci
+on:
+  pull_request:
+    branches: [master]
+  push:
+    branches: [master]
+  # schedule:
+  # - cron: '00 01 * * *'
+jobs:
```

Comment:
@khorolets I've removed `matrix` as it doesn't seem necessary now that we don't have `Cross`, let me know if this was incorrect and I'll add back in.

Reply:
Fully agree